src/core/actions/index.ts

```diff
@@ -164,6 +185,7 @@ namespace Actions {
   export type RefreshState = Action<typeof REFRESH_STATE, any>;
   // added automatically by middleware to interact with redux-undo
   export const SAVE_STATE = 'SAVE_STATE';
+  // tslint:enable max-line-length
```

Comment:
I disabled max-line-length for the entire action declaration block, since there were maybe 10 or so individual `tslint:disable-next-line max-line-length` comments in there. I figured line length isn't important in a large declaration block like that. I just enabled it here to have the rule apply later in the file.